src/java/powerofnumber/PowerOfNumber.java

```diff
@@ -0,0 +1,20 @@
+package java.powerofnumber;
+
+public class PowerOfNumber {
+
+    private int number;
+    private int power;
+
+    public PowerOfNumber(Request request) {
```

Comment:
it is bad idea to pass Request as a parameter

Reply:
in this case the class of the lower level depends on the class of the upper one
just pass two int parameter then Request request